producer/pdu_session.go

```diff
@@ -412,9 +421,8 @@ func HandlePDUSessionSMContextUpdate(smContextRef string, body models.UpdateSmCo
 				DLPDR.FAR.ApplyAction.Buff = true
 				DLPDR.FAR.ApplyAction.Nocp = true
 				smContext.PendingUPF[ANUPF.GetNodeIP()] = true
+				farList = append(farList, DLPDR.FAR)
```

Comment:
```go
farList = append(farList, DLPDR.FAR)
sendPFCPModification = true
smContext.SMContextState = smf_context.PFCPModification
```

Reply:
Right.
Also, as with the comment above, wouldn't it be clearer to http rsp here?
```go
farList = append(farList, DLPDR.FAR)
sendPFCPModification = true
smContext.SMContextState = smf_context.PFCPModification
smContext.SetState(smf_context.Active)
return &http_wrapper.Response{
	Status: http.StatusOK,
	Body:   response,
}
```

Reply:
no, it need to wait for pfcp response later.

Reply:
If DLPDR == nil, return http rsp.